pyccel/codegen/printing/cucode.py

```diff
@@ -0,0 +1,77 @@
+# coding: utf-8
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+"""
+Provide tools for generating and handling CUDA code.
+This module is designed to interface Pyccel's Abstract Syntax Tree (AST) with CUDA,
+enabling the direct translation of high-level Pyccel expressions into CUDA code.
+"""
+
+from pyccel.codegen.printing.ccode import CCodePrinter, c_library_headers
+
+from pyccel.ast.core        import Import, Module
+
+from pyccel.errors.errors   import Errors
+
+
+errors = Errors()
+
+# TODO: add examples
+
+__all__ = ["CudaCodePrinter"]
+
+class CudaCodePrinter(CCodePrinter):
+    """
+    Print code in CUDA format.
+
+    This printer converts Pyccel's Abstract Syntax Tree (AST) into strings of CUDA code.
+    Navigation through this file utilizes _print_X functions,
+    as is common with all printers.
+
+    Parameters
+    ----------
+    filename : str
+            The name of the file being pyccelised.
+    prefix_module : str
+            A prefix to be added to the name of the module.
+    """
+    printmethod = "_cucode"
```

Comment:
Do you know what this property is for?

Reply:
I assumed that as a way to make the name of the module unique if there is any name conflicts.
It was in the `CCodePrinter` class so I assumed that it was useful.

Reply:
I can't find it used anywhere

Reply:
oh I thought you were talking about `prefix_module`

`printmethod`
 removed 👍 [d935f95](https://github.com/pyccel/pyccel-cuda/pull/32/commits/d935f95ed675a85c8bb63ad264b8e8e692290282)